daffodil-test/src/test/resources/org/apache/daffodil/section02/validation_errors/Validation.tdml

```diff
@@ -643,7 +685,63 @@
 
 		</tdml:validationErrors>
   </tdml:parserTestCase>
-  
+
+    <tdml:parserTestCase name="validation_testFacets_01"
+                         root="e2_6" model="TestFacets"
+                         description="validates hexBinary against min/maxLength facet"
+                         validation="limited">
+
+    <tdml:document>
+        <tdml:documentPart type="byte">deadbeefdafada</tdml:documentPart>
+    </tdml:document>
+
+        <tdml:infoset>
+            <tdml:dfdlInfoset>
+                <ex:e2_6>DEADBEEFDAFADA</ex:e2_6>
+            </tdml:dfdlInfoset>
+        </tdml:infoset>
+
+    </tdml:parserTestCase>
+
+    <tdml:parserTestCase name="validation_testFacets_02"
+                         root="e2_7" model="TestFacets"
+                         description="validates hexBinary against length facet"
+                         validation="limited">
+
+    <tdml:document>
+        <tdml:documentPart type="byte">deadbeefdafada</tdml:documentPart>
+    </tdml:document>
+
+        <tdml:infoset>
+            <tdml:dfdlInfoset>
+                <ex:e2_7>DEADBEEFDAFADA</ex:e2_7>
+            </tdml:dfdlInfoset>
+        </tdml:infoset>
+    </tdml:parserTestCase>
+
+    <tdml:parserTestCase name="validation_testFacets_03"
+                         root="e2_8" model="TestFacets"
+                         description="checks that int doesn't work with length facet"
```

Comment:
Either this description or the entire test is incorrect since `e2_8` is really a `hex_length_0`.